itests/org.jboss.tools.cdk.ui.bot.test/src/org/jboss/tools/cdk/ui/bot/test/server/adapter/MinishiftServerAdapterTest.java

```diff
@@ -0,0 +1,57 @@
+/******************************************************************************* 
+ * Copyright (c) 2018 Red Hat, Inc. 
+ * Distributed under license by Red Hat, Inc. All rights reserved. 
+ * This program is made available under the terms of the 
+ * Eclipse Public License v1.0 which accompanies this distribution, 
+ * and is available at http://www.eclipse.org/legal/epl-v10.html 
+ * 
+ * Contributors: 
+ * Red Hat, Inc. - initial API and implementation 
+ ******************************************************************************/
+package org.jboss.tools.cdk.ui.bot.test.server.adapter;
+
+import org.eclipse.reddeer.junit.runner.RedDeerSuite;
+import org.jboss.tools.openshift.reddeer.requirement.CleanOpenShiftExplorerRequirement.CleanOpenShiftExplorer;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * Minishift server adapter's basic functionality test class
+ * @author odockal
+ *
+ */
+@CleanOpenShiftExplorer
+@RunWith(RedDeerSuite.class)
+public class MinishiftServerAdapterTest extends CDKServerAdapterAbstractTest {
```

Comment:
Does this class run individually? I am using in tests also @OpenPerspective(Jboss) to have more stable tests...

Reply:
Probably not, but I prefer to not restrict this option in case you need it run individually. How does @openperspective(Jboss) influence test stability?

Reply:
If you running tests not against devstudio, but against eclipse - e.g. from maven without profile download-devstudio or if you using eclipse with jboss tools target platform